app/src/models/github-repository.ts

```diff
@@ -52,6 +52,11 @@ export class GitHubRepository {
   }
 }
 
+export type ForkedGitHubRepository = GitHubRepository & {
```

Comment:
I would love it if we could document this similar to how we've documented `RepositoryWithGitHubRepository`